File: C++/freeCodeCamp.org_C++ Tutorial for Beginners/src/021_2d Arrays & Nested Loops/TwoDimensionalArraysAndNestedLoops.cpp
#include <iostream>

/* Simple C++ program that uses twodimensional arrays an nested for loops */

using namespace std;

int main()
{
    int numberGrid[3][2] = { // This is a two dimensional matrix
        {1, 2},
        {3, 4},
        {5, 6}
    };

    for(int i = 0; i < 3; i++) // This loop iterates over the elements in the first array
    {
        for(int j = 0; j < 2; j++) // This nested loop iterates over the elements in the second array
        {
            cout << numberGrid[i][j] << "\t";
        }
        cout << endl << endl;
    }

    return 0; 
}

File: Python/002_OOP Basics/021_2d Arrays & Nested Loops/TwoDimensionalArraysAndNestedLoops.py
import numpy # import numpy

# Python program that uses twodimensional arrays an nested for loops
numberGrid = numpy.array([ #  Using the numpy.array() function to create a two dimensional matrix
    [1, 2],
    [3, 4],
    [5, 6]
])

print() # newline

for i in range(len(numberGrid)): # This loop iterates over the elements in the first array

    for j in range(len(numberGrid[i])): # This nested loop iterates over the elements in the second array
        print(numberGrid[i][j])

    print()